test/javascripts/acceptance/preferences-test.js

```diff
@@ -427,3 +427,43 @@ QUnit.test("can select an option from a dropdown", async (assert) => {
   await field.selectRowByValue("Cat");
   assert.equal(field.header().value(), "Cat", "it sets the value of the field");
 });
+
+acceptance("User Preferences disabling dark mode", {
+  loggedIn: true,
+  settings: { default_dark_mode_color_scheme_id: 1 },
+});
+
+QUnit.test("shows option to disable dark mode", async (assert) => {
+  await visit("/u/eviltrout/preferences/interface");
+  assert.ok(
+    $(".control-group.dark-mode").length,
+    "it has the option to disable dark mode"
+  );
+});
+
+acceptance("User Preferences disabling dark mode", {
+  loggedIn: true,
+  settings: {
+    top_menu: "categories|latest|top|bookmarks",
+  },
+});
```

Comment:
I am not sure why all this dark mode stuff is here, possibly merge/rebase shenanigans?

Reply:
Good catch.  Yes a rebase issue.  Will address and revert.

Reply:
@martin-brennan removed the imposter test and embellished the commit message to better describe the change.  thanks for your review!